tests/VolleyManagement.UnitTests/Services/TournamentService/TournamentServiceTests.cs

```diff
@@ -1363,6 +1363,80 @@ public void Archive_AnyState_AuthorizationCheckInvoked()
 
         #endregion
 
+        #region Activate
+
+        /// <summary>
+        /// Activate Tournament by id.
+        /// 'CheckAccess' is invoked.
+        /// </summary>
+        [TestMethod]
+        public void Activate_AnyState_CheckAccessInvoked()
+        {
+            // Arrange
+            MockGetByIdQuery(new TournamentBuilder().Build());
+            var sut = BuildSUT();
+
+            // Act
+            sut.Activate(FIRST_TOURNAMENT_ID);
+
+            // Assert
+            VerifyCheckAccess(AuthOperations.Tournaments.Activate, 
+                Times.Once());
+        }
+
+        /// <summary>
+        /// Activate Tournament by id.
+        /// Tournament exists and it's being activated.
+        /// </summary>
+        [TestMethod]
+        public void Activate_TournamentExists_CommitInvoked()
```

Comment:
Are you testing that commit is invoked? Or that tournament has been activated?

Reply:
Commit invoked.